_scripts/instruction-widget/templates/install/centos.html

```diff
@@ -16,46 +16,20 @@ <h3>Supported architectures:</h3>
     You'll need to enable the EPEL (Extra Packages for Enterprise Linux)
     repository.<br>
     Follow these instructions at
-    <a href='https://fedoraproject.org/wiki/EPEL#How_can_I_use_these_extra_packages.3F'>
+    <a href='https://fedoraproject.org/wiki/EPEL#Quickstart'>
       the Fedora wiki to enable EPEL</a>.
     </p>
     <p class="centered">
-    <a class="link-button" href='https://fedoraproject.org/wiki/EPEL#How_can_I_use_these_extra_packages.3F'>
+    <a class="link-button" href='https://fedoraproject.org/wiki/EPEL#Quickstart'>
       enable EPEL</a>
     </p>
 </li>
-<li>
-    Enable the optional channel
```

Comment:
Is this no longer needed as well? It sounds like it's maybe still needed for 7?

Reply:
It's still needed and the other packages listed at https://fedoraproject.org/wiki/EPEL#Quickstart for RHEL 7 may be needed now or in the future as well.

At least for now, I'd personally rather just link to their instructions which should always stay up to date rather than trying to duplicate them here.

Reply:
Makes sense. Could we add a note to the effect of "make sure to follow all instructions for your system, including installing optional packages and extra if necessary"?

Reply:
Good idea. The `PowerTools`/`codeready-builder` repositories are currently needed for the Certbot packages on RHEL 8 so I made the wording a little more generic, but otherwise added the suggestion.